gap/perm.gi

```diff
@@ -347,24 +347,44 @@ end;
 #! @BeginChunk ThrowAwayFixedPoints
 #! This method defines a homomorphism of a permutation group
 #! <A>G</A> to the action on the moved points of <A>G</A> if
-#! <A>G</A> does not have too many moved points. In the current setup, the
-#! homomorphism is defined if the number <M>k</M> of moved
+#! either <A>G</A> has a lot of fixed points, or if <A>G</A> knows that it is
+#! primitive and has fixed points. If <A>G</A> has fixed points but does not
+#! know whether it is primitive, then it returns <K>NotEnoughInformation</K>
+#! so that it may be called again at a later time.
```

Comment:
This section confused me a bit. How about something more like this?
```suggestion
#! <A>G</A> has any fixed points, and is either known to be primitive or the
#! ratio of fixed points to moved points exceeds a certain threshold. If <A>G</A>
#! has fixed points but does not meet either of these criteria, then it returns
#! <K>NotEnoughInformation</K> so that it may be called again at a later time.
```